src/components/Model3DViewer/Model3DViewer.tsx

```diff
@@ -516,6 +518,11 @@ export class Model3DViewer extends React.Component<Model3DViewerProps> {
   }
 
   private highlightNodes() {
+    const { highlightNodes } = this.props;
+    if (!highlightNodes) {
+      return;
```

Comment:
more up to @mrtraser but I believe that it's not very according to a clean code that this function doesn't always highlight nodes. So for me you should either put that logic `if (highlightNodes) { call highlightNodes () }` outside or rename the function to something like `highlightNodesIfAllowed`

Reply:
@mrtraser - what do you think?

Reply:
As I mentioned above, you definitely need to change method name.  About naming – `highlightMappedNodes` is ok for me.

Reply:
Changed:

- Property name to `highlightMappedNodes`
- Private method name to `highlightMappedNodesIfAllowed `
- Updated property description to `enable default highlighting of 3D nodes that are associated with some asset`